pallets/messages/src/tests/other_tests.rs

```diff
@@ -668,7 +668,7 @@ fn migration_to_v2_should_work_as_expected() {
 
 		let old_count = v2::old::Messages::<Test>::iter().count();
 		let new_count = MessagesV2::<Test>::iter().count();
-		let current_version = MessagesPallet::current_storage_version();
+		let current_version = MessagesPallet::in_code_storage_version();
```

Comment:
We need to ensure these are exactly the same values. If not it would brick the chain.

Reply:
Thanks for pointing this out. I forgot to include it in the notable changes section of the PR. It appears to be a rename, and I’ll make sure to add it to the notable changes.

https://github.com/paritytech/polkadot-sdk/pull/1554/files#diff-01dc4f43df9baa537f30c6b369525715d596a3068944f38458e9f160d5412d58R306